modules/op-supported-parameters-tekton-chains-configuration.adoc

```diff
@@ -24,7 +24,7 @@ Cluster administrators can use various supported parameter keys and values to co
 | `artifacts.taskrun.storage`
 | The storage backend for task run signatures. You can specify multiple backends as a comma-separated list, such as `“tekton,oci”`. To disable storing task run artifacts, provide an empty string `“”`.
 | `tekton`, `oci`, `gcs`, `docdb`, `grafeas`
-| `tekton`
+| `oci`
```

Comment:
I think we should highlight somehow that the default value will be `oci`
wdyt @jkhelil

Reply:
Yes +1

Reply:
I suppose the highlighting will be in release notes? That would make the most sense to me, WDYT? @PuneetPunamiya @jkhelil

Reply:
Yeah that sounds good as well 👍🏻